docs/docusaurus/src/css/left_navigation.scss

```diff
@@ -76,3 +76,27 @@ nav.menu {
   height: 0.75rem;
   width: 0.75rem;
 }
+.menu__link--sublist {
+  position: relative;
+}
+
+li.menu__list-item {
+  .menu__link--sublist-caret:after {
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%) rotateZ(0);
+    left: 0;
```

Comment:
Can we move it to the far right if we change this to right?

Reply:
We can delete all this code in order to move them to the right, as that's the default position docusaurus provides for dropdowns. I'm anyway leaving them on the left because of consistency, as we discussed in Slack. Let me know if you want to talk about this further!